lang/CMakeLists.txt

```diff
@@ -252,7 +252,25 @@ endif()
 
 if (SC_ABLETON_LINK)
 	message(STATUS "Compiling with Ableton Link support")
-	include(../external_libraries/link/AbletonLinkConfig.cmake)
+	if(SYSTEM_ABLETON_LINK)
+		find_path(ABLETON_LINK_INCLUDE ableton Link.hpp)
+		add_library(Ableton::Link IMPORTED INTERFACE)
```

Comment:
confused... don't we need to link to the actual library? this just looks like it puts the appropriate header in the include path

Reply:
never mind, i see it's a header-only library